### DISCUSSION TITLE
OTP is not receiving

### DISCUSSION THREAD

**Discussion by NivethaE14** [upyog/UPYOG#1116] [Q&A]
while login we are not getting otp what should be done for that? even default otp is not working after user pod port forward step also.

**Comment by vinothrallapalli-eGov** [comment]
Please refer this discussion

https://github.com/orgs/upyog/discussions/1113#discussioncomment-9578497

**Reply by NivethaE14** [reply]
![Screenshot (47)](https://github.com/upyog/UPYOG/assets/166114393/7f8341d2-6873-4579-a724-ceb91f996d4c)
It is not working what should be done for this?
